RandomizerCore/ROM.cs

```diff
@@ -1189,6 +1189,37 @@ public void DashSpell(Assembler asm)
         a.Byt(dash);
     }
 
+    public void CombineFireSpell(Assembler asm, List<Collectable>? customSpellOrder, Random RNG)
+    {
+        // These are the bit flags for each spell.  The old implementation
+        // changed this table directly. However as this table is also used in
+        // the BIT comparison operation when determining if a spell should be
+        // cast, it lead to inconsistent behavior where some spells would be
+        // two-way linked and some would not.
+        byte[] spellBytes = GetBytes(0xdcb, 8);
```

Comment:
feels a bit weird reading this table from ROM when its just a power of two table ie: `2^0, 2^1, 2^2` etc
i don't have a preference on whether to keep it or change it, just pointing it out.

Reply:
I can see arguments for both. It becomes clear where the data comes from here. If we just define that sequence and someone reads it, they might ask where it came from.

Reply:
```suggestion
        byte[] spellBytes = Enumerable.Range(0, 8).Select(i => (byte)(1 << i)).ToArray();
```

is a bit longer. Is it preferable?